app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md

```diff
@@ -55,150 +55,103 @@ keywords:
 
-* are an Irish citizen
-* have settled or pre-settled status under the [EU Settlement Scheme (EUSS)](https://www.gov.uk/settled-status-eu-citizens-families)
-* have [indefinite leave to remain](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk)
-
-If you are currently studying in the UK on a student visa, you may be able to apply for a [graduate visa](#graduate-visa).
-
-### EU Settlement Scheme
-
-If you or your family are from the European Economic Area (EEA) or Switzerland, and you or a family member were living in the UK by 31 December 2020, you may still be able to apply to the [EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families).
+#### Step 2
+Apply for a teaching job with a school which can sponsor your visa. Your salary must meet the requirements for a Skilled Worker visa. 
 
-### Check your status
-
-If you’re not sure about your immigration status, you can [check if you need a UK visa](https://www.gov.uk/check-uk-visa
-).
-
-## Skilled worker visa
+[Learn more about applying for a teaching job in England](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
 
-The main visa route for non-UK citizens wishing to teach in England is the skilled worker visa.
+#### Step 3
+When you have a confirmed offer of a job, you’ll be able to apply for your [Skilled Worker visa](https://www.gov.uk/skilled-worker-visa).  
 
-You’ll be able to apply for your skilled worker visa if:
+You will need to show: 
 
-* you have a job offer from a school that is a Home Office licensed visa sponsor 
-* you have a certificate of sponsorship from this school, confirming they have offered you a job and will sponsor you
+* you have a certificate of sponsorship from the school, confirming they have offered you a job and will sponsor you  
 * [you meet certain standards in speaking, reading, writing and understanding English](https://www.gov.uk/skilled-worker-visa/knowledge-of-english)
-* your salary meets the requirement for a skilled worker visa
+* your salary meets the requirement for a Skilled Worker visa
 
-[Learn more about applying for a teaching job in England](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
 
-### Salary requirement for a skilled worker visa
+### Salary requirement for a Skilled Worker visa
  
-The salary you must be paid to get a visa varies according to  whether you are employed as a ‘qualified’ or ‘unqualified’ teacher. This depends on your level of experience and qualifications – your employing school can explain whether the role you are applying for is qualified or unqualified. The salary requirement also varies depending on the region you are applying to work in.
+The salary you need to get a visa depends on if you're employed as a qualified or unqualified teacher.
+
```

Comment:
Is there an extra space here? Not sure how to delete it!

Reply:
Looks OK in the raw file - suggest leaving this as is